src/controllers/userController.js

```diff
@@ -1,28 +1,39 @@
 import { PrismaClient } from '@prisma/client'
+import validateEmail from '../validation/EmailVailidation.js'
+import validateNumber from '../validation/PhoneValidation.js'
 
 const prisma = new PrismaClient()
 
 export const createUser = async function (req, res) {
   let data = req.body
   console.log(data)
-
-  const newUser = await prisma.user.create({
-    data
-  })
-
-  // console.log(newUser)
-  res.status(201).send({ status: true, newUser })
+  if (!validateEmail(data.email)) {
+    res.status(400).send({ status: false, message: 'Enter valid email' })
+  } else if (!validateNumber(data.phone)) {
+    res.status(400).send({ status: false, message: 'Enter valid phone number' })
+  } else {
+    const newUser = await prisma.user.create({
+      data,
+    })
+    res.status(201).send({ status: true, newUser })
+  }
 }
 
 export const updateUser = async (req, res) => {
-  const { email, username, phone } = req.params
-  const user = await prisma.post.updateMany({
-    where: { email, username, phone },
-    data: {
-      email: req.body,
-      username: req.body,
-      phone: req.body
-    }
-  })
-  res.json(user)
+  const { email, username, phone } = req.body
+  if (!validateEmail(email)) {
+    res.status(400).send({ status: false, message: 'Enter valid email' })
+  } else if (!validateNumber(phone)) {
+    res.status(400).send({ status: false, message: 'Enter valid phone number' })
+  } else {
```

Comment:
Else condition can be an error here, we are assuming it will be a success. Also return generic error message for users